jsonschema2pojo-cli/src/main/scripts/jsonschema2pojo.bash

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
What's the reason for this script? We already have the `.sh` :thinking:

Reply:
It uses pushd/popd. If you put those in the sh. IDE with inspections will warn that those commands are not POSIX compliment.

Could I just remove the sh one?

Reply:
I have removed the bash version.